### ISSUE TITLE
[mesheryctl] Unable to remove Meshery operator deployments on `mesheryctl system stop`

### ISSUE THREAD

**Issue by navendu-pottekkat** [meshery/meshery#4773]
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Current Behavior

Users can deploy just the Meshery Operator in a cluster without installing Meshery on it (multi-cluster environments, out of cluster deployment of Meshery).

`mesheryctl` handle the uninstallation of these deployments as well when stopping Meshery.

When running `mesheryctl system stop` it throws an error:

```
✗ mesheryctl system stop  
Meshery deployments will be deleted from your cluster. Are you sure you want to continue [y/n]? y
Stopping Meshery...
Error: failed to stop Meshery
See https://docs.meshery.io/reference/mesheryctl/system for usage details
: cannot stop Meshery: uninstall: Release not loaded: meshery: release: not found
```

This line fails and only the CRDs get removed.

https://github.com/meshery/meshery/blob/5fd505bf682245da965b81e138d2d8096f17d7de/mesheryctl/internal/cli/root/system/stop.go#L148

#### Desired Behavior

`mesheryctl system stop` should remove Meshery or any of its components completely from the cluster.

---

#### Contributor Resources
- [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

**Comment by navendu-pottekkat** [comment]
Is this because we also try to remove Meshery deployments and not just the operator deployments?

Should we check this in `mesheryctl` and use the `meshery-operator` charts for uninstalling if Meshery server is not deployed? Because, in this scenario, the Meshery Operator was deployed by Meshery Server using the `meshery-operator` charts.

// @Aisuko @panyuenlau @DelusionalOptimist @leecalcote

**Comment by panyuenlau** [comment]
@navendu-pottekkat How did you install the meshery-operator in your environment?

**Comment by navendu-pottekkat** [comment]
@panyuenlau I missed this. Sorry for the late reply.

I ran Meshery locally and the Meshery server deployed the Meshery Operator. I then tried stopping with `mesheryctl` in Kubernetes platform (which will use both the Helm charts (operator and Meshery)) and it threw the error.

Is there a straightforward solution to this. The one I have in mind is in the above comment. I am not sure if we can handle this more gracefully.
